vendor/plugins/edu.harvard.dce.paella.hideableExtendedTabAdapterPlugin/adapter.js

```diff
@@ -3,37 +3,62 @@
  * Modified to be smaller, whiter, and not display if nothing is using it.
  * Reference: es.upv.paella.extendedTabAdapterPlugin
  */
-Class ("paella.plugins.extendedTabAdapterPlugin",paella.ButtonPlugin,{
+Class ("paella.plugins.extendedTabAdapterPlugin", paella.ButtonPlugin, {
```

Comment:
These changes are just formatting artifacts. The "hideableExtendedTabAdapterPlugin" is no longer enabled in this pull, because the handouts are moved to the info plugin.